packages/integrations/svelte/src/index.ts

```diff
@@ -78,13 +73,12 @@ export default function (options?: Options | OptionsCallback): AstroIntegration
 		name: '@astrojs/svelte',
 		hooks: {
 			// Anything that gets returned here is merged into Astro Config
-			'astro:config:setup': ({ command, updateConfig, addRenderer, config }) => {
+			'astro:config:setup': ({ command, updateConfig, addRenderer }) => {
 				addRenderer(getRenderer());
 				updateConfig({
 					vite: getViteConfiguration({
 						options,
 						isDev: command === 'dev',
-						postcssConfig: config.style.postcss,
```

Comment:
I didn't add a changeset for the svelte integration as it should be overidden by https://github.com/withastro/astro/pull/5685 specifically [this change](https://github.com/withastro/astro/pull/5685/files#diff-03dddc60579af21eb48fb3156c2466b43ac2e80bd6cdb5a399b271d77336cbc1)